packages/histoire-controls/histoire.config.ts

```diff
@@ -0,0 +1,8 @@
+import { defineConfig } from 'histoire'
```

Comment:
The book moved to `histoire-controls-stories`

Reply:
Should have seen it *facepalm* 😅... Removed here: https://github.com/histoire-dev/histoire/pull/310/commits/67c61e29a3b9e75c71f4a06c2ae1e29712259a99